picard/ui/coverartbox.py

```diff
@@ -201,8 +200,21 @@ def show(self):
             self._show()
         QtGui.QGroupBox.show(self)
 
+    def _extract_common_metadata(self, item):
+        related_files = [track.linked_files[0] for track in item.tracks
+                         if track.num_linked_files == 1]
+        image_hashes_list = [[image.datahash for image in related_file.orig_metadata.images]
```

Comment:
Using a list of sets would be prolly faster in the `all()` equality test. Also it would prevent order of elements to have any impact (not sure if we have any guarantee about order of images in orig_metadata.images, but we don't care order here anyway).
What do you think ?